pkg/client/cache/listers.go

```diff
@@ -123,6 +123,61 @@ func (s *StoreToNodeLister) GetNodeInfo(id string) (*api.Node, error) {
 	return minion.(*api.Node), nil
 }
 
+// StoreToControllerLister gives a store List and Exists methods. The store must contain only ReplicationControllers.
+// Note that this lister matches the given selectors against the rcs Spec.Selectors, not its Spec.Labels.
```

Comment:
List now takes no arguments because I only use the following:
1. GetPodControllers for reverse lookup
2. GetByKey to lookup controllers